examples/model.py

```diff
@@ -96,3 +105,35 @@ def forward(
         )
 
         return self.head(x_dict[entity_table][: seed_time.size(0)])
+
+    def forward_id_awareness(
```

Comment:
I think `forward_id_awareness` is confusing. I assume you mean `forward_rhs_readout` or something like that.

Reply:
Sounds good.